.travis.yml

```diff
@@ -32,6 +32,7 @@ install:
     pip install --upgrade setuptools
     pip install $NUMPY
     pip install $MPL
+    pip install -r requirements.txt
   - |
     cd geos-3.3.3
     export GEOS_DIR=$HOME/.local/
```

Comment:
I assume compiling GEOS isn't a huge issue.

Migrating to shapely should be done at a later point in time.